DSP_Plugins.GalacticScale/Scripts/PatchPlanetSize/PatchNearColliderLogic.cs

```diff
@@ -0,0 +1,127 @@
+﻿using HarmonyLib;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using Patch = GalacticScale.Scripts.PatchPlanetSize.PatchForPlanetSize;
+
+namespace GalacticScale.Scripts.PatchPlanetSize
+{
+    [HarmonyPatch(typeof(NearColliderLogic))]
+    static class PatchNearColliderLogic
+    {
+        public static int offset = -20;
+        public static List<GameObject> markers;
+
+        [HarmonyPostfix]
+        [HarmonyPatch("Update")]
+        public static void Update()
+        {
+            int o = offset;
+            if ((Input.GetKeyUp(KeyCode.Equals) || Input.GetKeyUp(KeyCode.KeypadPlus)) && VFInput.alt)
```

Comment:
what is it for ?

Reply:
Increases and Decreases the vein search area node size, allows you to use alternative miners